### PR TITLE
Bump version to support basic auth

### PR DESCRIPTION
**Pull request by suyograo**
Thanks for contributing to Logstash! If you haven't already signed our CLA, here's a handy link: https://www.elastic.co/contributor-agreement/

### CODE REVIEW

**Comment by andrewvc** [comment]
Thanks for adding this! Before we merge I think we need to steal the docs from https://github.com/logstash-plugins/logstash-input-http_poller/pull/69

**Comment by suyograo** [comment]
+1. will add docs

File: logstash-output-http.gemspec [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   # Gem dependencies
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
-  s.add_runtime_dependency "logstash-mixin-http_client", ">= 2.2.1", "< 5.0.0"
+  s.add_runtime_dependency "logstash-mixin-http_client", ">= 5.1.0"
```
Comment (andrewvc) [inline logstash-output-http.gemspec]: Hmmm, we don't want this grabbing a `> 6.0.0` version though.

Comment (suyograo) [inline logstash-output-http.gemspec]: I'll add the constraint now

**Pull request closed without merging**